### ISSUE TITLE
Employee IP agreement for Germany / Europe

### ISSUE THREAD

**Issue by penguineer** [github/balanced-employee-ip-agreement#53]
Thanks for creating these documents, I really like the idea!

I would very much love to see a version that applies to Germany and/or Europe, however, I do not have the necessary legal knowledge to do the transformation. (There was a slight understanding on Twitter yesterday: I do not need a translation in order to understand the document, but in the current state it most likely does not apply.)

As there are many common regulations in Europe, I guess the solution can be as follows:
* exchange the regulations by those applicable for Europe
* add an addendum for each country (as done with the states)
* get translations for the specific countries

As FLOSS is on the European political agenda, maybe the EU Administration would even sponsor the translations?

I'm sorry that I cannot contribute more than the issue and my enthusiasm. Hopefully somebody is as motivated and more qualified. :)

**Comment by another-guy** [comment]
Small note before I comment: _Yes, I **did** read the disclaimer section_.

I have a similar question about whether the translation(s) into Russian (and other languages) would be possible and valid in terms of legality.

As far as my understanding goes, the laws on intellectual property may or does vary from country to country. This probably makes finding the "one size fits all" contract definitions and wording a very tricky problem.

_So, are translation considered to be a part of the roadmap or this repo is more of an USA-relevant "project" with no plans of "growth", so to speak?_

In my humble opinion, the entire international software development community deserves to have this sort of document for protecting employee's rights. Or, put a slightly different way, the community deserves a standardized form of a contract that regulates the IP rights distribution much more fairly than it is today. Of course, it's not just about developers -- many specialists in other fields should have a clear way to separate their own stuff from the employers' one.

**Comment by kemitchell** [comment]
@another-guy, a Russian analog to this kind of agreement would interest me greatly. But I doubt  translating this particular agreement into the Russian language, and then into the Russian legal context, would be efficient or worthwhile. Better to start from a blank page, looking to this for general inspiration, with a Russian юрисконсульт who works with small Russian companies or startup initiatives.

That does not mean this form must be US-specific. If you had asked about adapting the agreement for Canada, Australia, New Zealand, Australia, South Africa, or England, my answer would be different. Both legally and linguistically, those jurisdictions are more compatible. Conversely, it might be quite possible to share and translate a common employee IP agreement for EU and CIS companies. I'm not sure.

It's a bit like porting system software. Agreements about IP and work have to fit within the limits of IP and work laws in the relevant country. Those laws are like an operating system: you can invoke their code with system calls, and get them to run code of your own within limits, but your code has to work with the OS. If your program is pretty low-level, it often makes more sense to start from scratch, with inspiration, than to port existing code.

**Comment by another-guy** [comment]
@kemitchell 

> Better to start from a blank page, looking to this for general inspiration, with a Russian юрисконсульт who works with small Russian companies or startup initiatives

I agree that would be the ideal way of attacking it. I don't see it happening in the near future since nobody owns the problem on Russia's side [yet].

> But I doubt translating this particular agreement into the Russian language, and then into the Russian legal context, would be efficient or worthwhile.

Intuitively I'm inclined to agree with you on that too. However, as I understand, most of the Russian codes including the Civil Code (Гражданский Кодекс), and even Constitution, are similar to the European/US ones for the historical reasons. The spirit and the letter of IP-related law should be pretty close when we compare specific Western and Russian normative acts. So, may be translation will be a pretty good _initial attempt_ here?

Now, IANAL, so my assumptions may be very very wrong. :)

**Comment by mlinksva** [comment]
It's certainly possible to do this, at least on a country-by-country basis. An example of "porting" IP licenses to _many_ jurisdictions is Creative Commons, prior to version 4.0 of its suite of licenses. Copyright law is similar enough across jurisdictions that they decided to [stop porting](https://wiki.creativecommons.org/wiki/Porting_Project) and rely on language-only translations for 4.0. But employment law is more variable, which will make it harder to have a single agreement that is merely translated, and maybe challenging to even "port" to some jurisdictions without starting from scratch as @kemitchell notes.

GitHub employs some people in non-US jurisdictions including Germany, and for these outside counsel have made changes necessary to comply with local law. We haven't gotten around to vetting whether we can release these and what it would take to make them reusable and as close to BEIPA as possible. No promises as to schedule, but we do hope to do that.

To help inform this discussion I too would love to hear from developers/knowledge workers and legal experts working in other jurisdictions about IP agreement practices and law that might inform such work toward versions of BEIPA that apply in Germany and/or Europe or elsewhere. More concrete and detailed is better, but high level overviews such as one [covering France, Germany, and UK](http://www.lexology.com/library/detail.aspx?g=d1d5db8f-f31c-43d8-8d49-43c7e42f7086) are interesting reading.

**Comment by another-guy** [comment]
@mlinksva Thanks for the update. You're totally right about the employment/labor law indeed are much more unalike across the world, as opposed to the IP law. Somehow I missed this point while writing my original comment, and only remembered it afterwards when it was too late.

I'll keep looking at this (and other) repo issues to not miss an important update in the future. And thanks for doing the great job!

**Comment by shushugah** [comment]
I would be very interested in a German friendly equivalent of this, let me know if something is in the works!

**Comment by mlinksva** [comment]
Some progress at https://github.com/github/balanced-employee-ip-agreement/pull/44#issuecomment-743601015
